lib/internal/modules/cjs/loader.js

```diff
@@ -766,6 +767,14 @@ Module._load = function(request, parent, isMain) {
   }
 
   const filename = Module._resolveFilename(request, parent, isMain);
+  if (StringPrototypeStartsWith(filename, 'node:')) {
+    const specifier = StringPrototypeSlice(filename, 5);
+
+    const mod = loadNativeModule(specifier, request);
+    if (mod && mod.canBeRequiredByUsers) return mod.exports;
```

Comment:
```suggestion
    if (mod?.canBeRequiredByUsers) return mod.exports;
```
nit: should we use optional chaining here?

Reply:
It'd be nice to backport that to all release lines though.